app/src/main/java/com/devshish/internship/model/Song.kt

```diff
@@ -0,0 +1,8 @@
+package com.devshish.internship.model
+
+data class Song(
+    val title: String,
+    val artist: String,
+    val duration: Int,
+    val image: String = ""
```

Comment:
I think it is not best practice to make default parameters in model,
and from model perspective it is better to use String? if imageUrl is absent